onnx_tensorrt/backend.py

```diff
@@ -51,16 +49,38 @@ def count_trailing_ones(vals):
 def _tensorrt_version():
     return [int(n) for n in trt.__version__.split('.')]
 
+# If TensorRT major is >= 5, then we use new Python bindings
+USE_PYBIND = _tensorrt_version()[0] >= 5
+
+if USE_PYBIND:
+    TRT_LOGGER = trt.Logger(trt.Logger.WARNING)
+
+if not USE_PYBIND:
```

Comment:
Why do we need two versions of python binding? Can we just use the pybind11 one for all trt version?

Reply:
@yinghai TensorRT5 requires pybind11 bindings.  TensorRT4 requires SWIG.
If we want to be backward compatible we need to support both. TensorRT4 is the done deal and IMHO nobody is going back to retrofit it for pybind11.

Reply:
@yinghai as @goldgeisser said having both pybind11 and SWIG bindings is necessary for onnx-trt to be backwards compatible with versions < 5.0. If this answer is satisfactory please accept this PR.